### PR TITLE
Add unit tests and snapshots for SVGIcon

### PR DESCRIPTION
**Pull request by simpleimpulse**
## Description

- Add unit tests
- Add snapshots

## Checklist before submitting pull request

- [x] New code is unit tested
- Commits follow our [Developer Guide](https://tds.telus.com/contributing/developer-guide.html#make-a-commit)
- [x] For code changes, run `npm run prepr` locally
  - make sure visual and accessibility tests pass

### CODE REVIEW

File: packages/DecorativeIcon/__tests__/SVGIcon.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  const doShallow = (props = { symbol: 'artificialIntelligence' }) =>
+    shallow(<SVGIcon {...defaultProps} {...props} />)
+
+  it('renders', () => {
+    const icon = render(<SVGIcon variant="default" symbol="artificialIntelligence" />)
+
+    expect(icon).toMatchSnapshot()
+  })
+
+  it('is hidden from screen readers', () => {
```
Comment (theetrain) [inline packages/DecorativeIcon/__tests__/SVGIcon.spec.jsx]: There should be unit tests for size and variant as well.

File: packages/DecorativeIcon/__tests__/SVGIcon.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    const icon = doShallow()
+    expect(icon).toHaveProp('aria-hidden', 'true')
+  })
+})
+
+describe('BatteryCharging', () => {
```
Comment (theetrain) [inline packages/DecorativeIcon/__tests__/SVGIcon.spec.jsx]: Rename this to 'SVGIcon consumer'

Comment (simpleimpulse) [reply packages/DecorativeIcon/__tests__/SVGIcon.spec.jsx]: Done

File: packages/DecorativeIcon/__tests__/SVGIcon.spec.jsx (hunk 1 above)
Comment (simpleimpulse) [reply packages/DecorativeIcon/__tests__/SVGIcon.spec.jsx]: Done

**Pull request merged**